src/main/java/frc/robot/RobotContainer.java

```diff
@@ -145,6 +151,23 @@ private void configureButtonBindings() {
     // Change the raw boolean to true to pick keyboard durring simulation
     boolean usingKeyboard = true && Robot.isSimulation();
 
+    OI.getTrigger(OI.Driver.RTrigger).whileTrue(intake.pivotDownCommand());
+    OI.getTrigger(OI.Driver.LTrigger).whileTrue(intake.pivotUpCommand());
+    OI.getTrigger(OI.Driver.DPAD_RIGHT).whileTrue(intake.intakeCommand());
+    OI.getTrigger(OI.Driver.DPAD_LEFT).whileTrue(intake.outtakeCommand());
+
+    // Set the intake rollers to the left and right triggers
+    OI.getPOVButton(OI.Driver.DPAD_UP)
+        .and(OI.getButton(OI.Driver.RBumper).negate())
+        .whileTrue(intake.intakeToBirdhouse());
+    OI.getPOVButton(OI.Driver.DPAD_UP)
+        .and(intake.getBeamBroken().negate())
+        .whileTrue(coralScorer.scoreClockWise());
+    OI.getPOVButton(OI.Driver.DPAD_DOWN)
+        .and(OI.getButton(OI.Driver.RBumper).negate())
+        .whileTrue(intake.ejectFromBirdhouse());
+    intake.setDefaultCommand(intake.retractPivotCommand());
+
```

Comment:
Idk if you saw @azvanderpas message in slack, but we should makes these non PID buttons that are for testing, only if Robot.isTesting() is true.

Reply:
Lines 154-157 should be Test mode. But the implementation is unfortunately not so simple. We'll have to think about the best way to remap buttons when the mode changes.